app/konnect/mesh-manager/service-mesh.md

```diff
@@ -34,54 +38,196 @@ After creating the global control plane, you must add a zone to that control pla
 
 You now have a very basic {{site.mesh_product_name}} service mesh added to {{site.konnect_short_name}}. This service mesh can only create meshes and policies at the moment, so you need to add services and additional configurations to it.
 
-## Install a demo service
+<!--This is standalone mode, right? Does standalone make sense with this tutorial? Is there a way we can bring that more into the narrative? Ex. In this scenario, we will be using standalone mode because ____-->
```

Comment:
This is standalone mode, right? Does standalone make sense with this tutorial? Is there a way we can bring that more into the narrative? Ex. In this scenario, we will be using standalone mode because ____. If it is standalone, why do we still need a zone? It looks like in Mesh you don't need one (just need Kuma CP?), does Konnect require it?